kyuubi-main/src/main/scala/org/apache/kyuubi/operation/KyuubiOperationManager.scala

```diff
@@ -178,6 +178,7 @@ class KyuubiOperationManager private (name: String) extends OperationManager(nam
         if (remoteHandle != null) {
           val or = FetchOrientation.toTFetchOrientation(order)
           val req = new TFetchResultsReq(remoteHandle, or, maxRows)
+          req.setFetchType(1.toShort)
```

Comment:
Is this related? can we rebase master branch?

Reply:
I mean you need rebase master branch

Reply:
@yaooqinn help, can we have a chat in Gitter?